greenmail-core/src/main/java/com/icegreen/greenmail/smtp/commands/RcptCommand.java

```diff
@@ -27,7 +27,7 @@
  */
 public class RcptCommand
         extends SmtpCommand {
-    static Pattern param = Pattern.compile("RCPT TO:\\s?<([^>]+)>",
+    static Pattern param = Pattern.compile("RCPT TO:\\s?<([^>]+[^\\s]+)>",

```

Comment:
Can you give an example?
The problem is that this changes the behaviour of the rexex. Previously "RCPT TO: <aa>>>>" didn't match the regex, now it does (and the capture group catches "aa>>>")